src/Engine.php

```diff
@@ -112,6 +112,13 @@ public function execute(int $threads): bool
 
     private function runInitialTestSuite(): void
     {
+        if ($this->config->shouldSkipInitialTestSuite()) {
```

Comment:
As I mentioned above, it should not be allowed to use `--skip-initial-test-suite` **without** `--coverage`.

Now it's possible:

<img width="975" alt="zsh 2020-02-16 18-11-33" src="https://user-images.githubusercontent.com/3725595/74607223-1436ec80-50e8-11ea-8813-b5bb1a35ae11.png">

The existing red message is confusing for this case. We need another one that will say exactly about this issue: that `--skip-initial-test-suite` option requires `--coverage`

Reply:
![image](https://user-images.githubusercontent.com/546811/74689532-6143bd00-51d3-11ea-9ee2-f85ff41bd42b.png)